### ISSUE TITLE
Cannot read property 'length' of undefined - In latest cli version - Again

### ISSUE THREAD

**Issue by ondrej-hegedus** [aws-amplify/amplify-cli#7211]
The latest version @aws-amplify/cli@4.50.0

raises the error
```
Cannot read property 'length' of undefined
An error occurred during the push operation: Cannot read property 'length' of undefined
```
without any additional data or at least the stacktrace.

version @aws-amplify/cli@4.49.0 works.
After upgrade it's impossible to deploy (amplify push).
The change in the code between those two push operations was a simple white-space.

I wonder if you would be sometime able to prevent this issues from happening in future versions.
This bugs were visible in the versions 2.x

Thanks

**Comment by cjihrig** [comment]
Thank you for reporting this @ondrej-hegedus. I'm going to close this as a duplicate of https://github.com/aws-amplify/amplify-cli/issues/7190. Please feel free to engage with that issue. There are also steps to get unblocked detailed there.
